apps/web/modules/videos/ai/ai-transcribe.tsx

```diff
@@ -26,11 +25,25 @@ export const CalAiTranscribe = () => {
   );
 
   useDailyEvent("transcription-started", (ev) => {
-    showToast(t("transcription_enabled"), "success");
+    daily?.updateCustomTrayButtons({
+      transcription: {
+        label: "Stop",
+        tooltip: "Toggle real time transcription powered by AI",
```

Comment:
```suggestion
        tooltip: "Transcription powered by AI",
```

<img width="321" alt="CleanShot 2024-05-17 at 11 02 57@2x" src="https://github.com/calcom/cal.com/assets/8019099/a240afa8-43da-4808-919f-95e76132d4ce">